### PR TITLE
Update/agenda cards header

### PR DESCRIPTION
**Pull request by scott062**
[API PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/765)

### CODE REVIEW

File: src/sass/_panelMeetingAgendas.scss [hunk 1]
```diff
@@ -120,8 +120,21 @@
...
+        margin: 10px 0px;
+        margin-right: 25%;
+        gap: 10px;
+        align-items: flex-start;
+        background: linear-gradient(white, white) padding-box, linear-gradient(to right, #d6d7d9, #d6d7d900) border-box;
+        border: 2px solid transparent;
+        border-top: none;
+        border-left: none;
+        border-right: none;
       }
```
Comment (scott062) [inline src/sass/_panelMeetingAgendas.scss:138]: I'm losing my mind with how to style this and still look good. Would prefer to toggle hide/expand the content or anything else b/c it doesn't size well with the rest of the card. But I know thats not an option. Let me know if you have any feedback here

File: src/Components/Agenda/AgendaItemRow/AgendaItemRow.jsx [hunk 2]
```diff
@@ -19,19 +21,24 @@ const AgendaItemRow = props => {
...
-  const userSkill = get(agenda, 'skill') || 'None Listed';
-  const userLanguage = get(agenda, 'language') || 'None Listed';
-  const userBureau = get(agenda, 'bureau') || 'None Listed';
-  const userGrade = get(agenda, 'grade') || 'None Listed';
+  const userSkill = <SkillCodeList skillCodes={get(clientData, 'skills') || []} />;
+  const userLanguage = get(clientData, 'languages') || [];
+  const userBureau = get(clientData, 'current_assignment.position.bureau') || 'None Listed';
+  const userGrade = get(clientData, 'grade') || 'None Listed';
+  const cdo = get(clientData, 'cdo.name') || 'None Listed';
+  console.log('clientData', clientData);
```
Comment (scott062) [inline src/Components/Agenda/AgendaItemRow/AgendaItemRow.jsx:41]: 🧼

File: src/Components/Agenda/AgendaItemRow/AgendaItemRow.jsx [hunk 3]
```diff
@@ -19,19 +21,24 @@ const AgendaItemRow = props => {
     isPanelMeetingView,
   } = props;
 
+  const isCard = false;
```
Comment (scott062) [inline src/Components/Agenda/AgendaItemRow/AgendaItemRow.jsx:24]: Need to go check this and come up with real solution for moved remarks

**Comment by scott062** [comment]
Need to check more universal collapse behavior on these cards as well 💀

**Comment by scott062** [comment]
Update fuse search for the new fields here or in separate PR?

File: src/Components/Agenda/AgendaItemRow/AgendaItemRow.jsx (hunk 3 above)
Comment (scott062) [reply src/Components/Agenda/AgendaItemRow/AgendaItemRow.jsx]: Done

File: src/sass/_panelMeetingAgendas.scss (hunk 1 above)
Comment (elizabeth-jimenez) [reply src/sass/_panelMeetingAgendas.scss:150]: image
what about margin-left 25% instead of right?

**Comment by elizabeth-jimenez** [comment]
> Update fuse search for the new fields here or in separate PR?

separate, cause we still need to know which languages, bureau, etc, they want to filter on, now that u have leg-based and person-based values

**Comment by elizabeth-jimenez** [comment]
that faded border 😸

File: src/sass/_panelMeetingAgendas.scss (hunk 1 above)
Comment (elizabeth-jimenez) [reply src/sass/_panelMeetingAgendas.scss:150]: nvm, messes up at smaller resolutions - looking into this

**Comment by elizabeth-jimenez** [comment]
Approved pending feedback: the overflow at smaller screens

File: src/sass/_panelMeetingAgendas.scss [hunk 4]
```diff
@@ -132,8 +132,22 @@
...
-        font-size: 1.4rem;
+        justify-content: flex-end;
+        font-size: 16px;
+        height: 19px;
+      }
+
+      .panel-meeting-person-data {
+        display: flex;
+        margin: 10px 0px;
+        margin-right: 25%;
```
Comment (elizabeth-jimenez) [inline src/sass/_panelMeetingAgendas.scss:143]: can combine

File: src/sass/_panelMeetingAgendas.scss [hunk 5]
```diff
@@ -132,8 +132,22 @@
...
+        gap: 10px;
+        align-items: flex-start;
+        background: linear-gradient(white, white) padding-box, linear-gradient(to right, #d6d7d9, #d6d7d900) border-box;
+        border: 2px solid transparent;
+        border-top: none;
+        border-left: none;
+        border-right: none;
```
Comment (elizabeth-jimenez) [inline src/sass/_panelMeetingAgendas.scss:150]: you should just be able to define border-bottom for the bottom line

File: src/sass/_panelMeetingAgendas.scss [hunk 6]
```diff
@@ -132,8 +132,22 @@
...
+        height: 19px;
+      }
+
+      .panel-meeting-person-data {
+        display: flex;
+        margin: 10px 0px;
+        margin-right: 25%;
+        gap: 10px;
+        align-items: flex-start;
+        background: linear-gradient(white, white) padding-box, linear-gradient(to right, #d6d7d9, #d6d7d900) border-box;
```
Comment (elizabeth-jimenez) [inline src/sass/_panelMeetingAgendas.scss:146]: very cool way to get that gradient border 😱

**Review by elizabeth-jimenez — APPROVED** [review]
💅

**Review by elizabeth-jimenez — DISMISSED** [review]
overflow

File: src/sass/_panelMeetingAgendas.scss (hunk 6 above)
Comment (scott062) [reply src/sass/_panelMeetingAgendas.scss:145]: Stack Overflow FTW 🙌

**Review by elizabeth-jimenez — APPROVED** [review]

**Pull request merged**